src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props

```diff
@@ -4,5 +4,6 @@
 		<_JavaInteropReferences>Java.Interop;System.Runtime</_JavaInteropReferences>
 		<DependsOnSystemRuntime Condition=" '$(DependsOnSystemRuntime)' == '' ">true</DependsOnSystemRuntime>
 		<CopyNuGetImplementations Condition=" '$(CopyNuGetImplementations)' == ''">true</CopyNuGetImplementations>
+		<_YieldDuringExecution Condition="'$(_YieldDuringExecution)' == ''">true</_YieldDuringExecution>
```

Comment:
The fully generic property (set to `true` by default in Common.targets) is [`YieldDuringToolExecution`](https://github.com/Microsoft/msbuild/blob/fdbf2d358238e8236383e81acb353937161dae29/src/Tasks/Microsoft.Common.CurrentVersion.targets#L71). I'd lean toward not providing an extra lever, but if you want to I'd go with the `Android` prefix to make it clear what it affects.